bblfsh/__main__.py

```diff
@@ -9,26 +9,55 @@ def setup():
     parser = argparse.ArgumentParser(
         description="Query for a UAST to Babelfish and dump it to stdout."
     )
+    parser.add_argument('filename', type=str,
```

Comment:
This change seems to be unrelated with the usage of `libuast`. Was it requested by the machine learning team?